Description: Add system path
 Resolve system modules
--- a/scripts/build-polyfills.js
+++ b/scripts/build-polyfills.js
@@ -18,11 +18,11 @@
   const bundle = await rollup.rollup({
     input: depName,
     plugins: [
-      commonjs(),
-      nodeResolve({
+        nodeResolve({
         browser: true,
         preferBuiltins: true
       }),
+      commonjs(),
       // license({
       //   thirdParty: {
       //     output: path.join(__dirname, '..', 'polyfills', `LICENSE-${depName}.txt`),
